recipes/openfe/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "openfe" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/OpenFreeEnergy/openfe/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 03b6531ed7c8b03f09294c9792ce465633c1d2e6aa60d0e34344a14b3dc56b96
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - "openfe = openfecli.cli:main"
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - click
+    - lomap2
+    - networkx
+    - numpy
+    - openff-toolkit
+    - plugcli
+    - python >=3.8
+    - rdkit
+    - typing-extensions
+
+test:
+  imports:
+    - openfe.setup
+    - openfe.orchestration
+    - openfe.simulation
+    - openfe.analysis
+  requires:
+    - pip
+  commands:
+    - pip check
```

```suggestion
    - pip check
    - openfe --help
```
This checks that the entry points are actually working. If `--help` is not available in the CLI, feel free to change it to something else.